usecases/scenarios/scenario_validation.go

```diff
@@ -207,6 +210,9 @@ func (self *ValidateScenarioIterationImpl) Validate(ctx context.Context,
 
 			result.SanctionCheck[idx].Query = queryValidation
```

Comment:
There is also a letover above of only checking for the `name` field, which will need to be generic now that we accept other fields.